test/format/format.h

```diff
@@ -296,6 +300,7 @@ typedef struct {
     bool lsm_config;                    /* At least one LSM data source configured */
     bool multi_table_config;            /* If configuring multiple tables */
     bool transaction_timestamps_config; /* If transaction timestamps configured on any table */
+    bool tiered_storage;                /* If tiered storage is configured */
```

Comment:
Alphabetize above `transaction_timestamps_config`
Do you want to make it consistent and end with a `_config` suffix? `tiered_config`?

Reply:
> The refactoring of testutil looks good to me. I see that the test format in evergreen is configured only for dir_store, Is there a seperate ticket for s3_store ?

@raviprakashgiri29, thanks for the nudge.  I just added WT-10796.  I think the plan will be to first make sure we have stable runs using `dir_store` before we tackle `s3_store`.

Reply:
Good idea, fixed.